lib/elixir/lib/calendar/iso.ex

```diff
@@ -1490,11 +1579,20 @@ defmodule Calendar.ISO do
 
   defp zero_pad(val, count) when val >= 0 do
     num = Integer.to_string(val)
-    :binary.copy("0", max(count - byte_size(num), 0)) <> num
+
+    case max(count - byte_size(num), 0) do
+      0 -> num
+      1 -> ["0", num]
+      2 -> ["00", num]
+      3 -> ["000", num]
+      4 -> ["0000", num]
+      5 -> ["00000", num]
+      6 -> ["000000", num]
```

Comment:
I don't think we would ever reach this, since byte_size is never 0. We should also check that `count <= 6`.